documentation/projects/proposals/additional_search_views/20230719-implementation_plan_additional_search_views.md

```diff
@@ -0,0 +1,263 @@
+search page and on the Additional search views. One thing that I'm not sure
+about here is whether the analytics events should stay the same and be sent from
+all pages - or if we should somehow differentiate the events coming from
+different search views.
+
+#### Add a `VCollectionHeader` component
+
+Figma links:
+[creator desktop](https://www.figma.com/file/niWnCgB7K0Y4e4mgxMrnRC/Additional-search-views?type=design&node-id=1200-56323&mode=design&t=pN0PPAzlKQEKT9tJ-4),
+[creator mobile](https://www.figma.com/file/niWnCgB7K0Y4e4mgxMrnRC/Additional-search-views?type=design&node-id=1200-56362&mode=design&t=suLIyJHNmZrM0mPH-4),
+[source desktop](https://www.figma.com/file/niWnCgB7K0Y4e4mgxMrnRC/Additional-search-views?type=design&node-id=1200-56381&mode=design&t=suLIyJHNmZrM0mPH-4),
+[source mobile](https://www.figma.com/file/niWnCgB7K0Y4e4mgxMrnRC/Additional-search-views?type=design&node-id=1200-56421&mode=design&t=suLIyJHNmZrM0mPH-4),
+[tag desktop](https://www.figma.com/file/niWnCgB7K0Y4e4mgxMrnRC/Additional-search-views?type=design&node-id=1216-58402&mode=design&t=suLIyJHNmZrM0mPH-4),
+[tag mobile](https://www.figma.com/file/niWnCgB7K0Y4e4mgxMrnRC/Additional-search-views?type=design&node-id=1200-56457&mode=design&t=suLIyJHNmZrM0mPH-4)
+
+The header should have an icon (tag, creator or source), the name of the
+tag/creator/source. For source and the creator, there should be an external link
+button.
+
+The header should also display the number of results, "251 audio files with the
+selected tag", "604 images provided by this source", "37 images by this creator
+in Hirshhorn Museum and Sculpture Garden".
+
+#### Add VCreatorLink component
+
+The "by creator" line under the main item on the single result page should be
+replaced with a `VCreatorLink` component - a wrapper over `VButton` with a link
+to the localized "creator" collection page.
+
+#### Add VSourceLink component
+
+Both the source and the provider should be replaced with `VSourceLink` component
+that links to the `source` page. This component is also a wrapper over `VButton`
+component.
+
+#### Update VMediaTag component to be a `VButton` wrapping a `VLink`
+
+Each `VMediaTag` should be a `VButton` with `as="VLink"` and should link to the
+localized page for the tag collection. Figma link:
+https://www.figma.com/file/niWnCgB7K0Y4e4mgxMrnRC/Additional-search-views?type=design&node-id=1200-56515&mode=design&t=nCX20BtJYqMOFAQm-4
+[VMediaTag component](https://github.com/WordPress/openverse/blob/c7b76139d5a001ce43bde27805be5394e5732d1a/frontend/src/components/VMediaTag/VMediaTag.vue)
+`<VButton variant="filled-gray" size="small" class="label-bold" as="VLink" href="href">{{tag}}</VButton>`.
+It should accept `href` as a prop.
+
+### Other interface changes
+
+#### Update the "by" area of the single result page
+
+The "by creator" line should be replaced with a section that is
+horizontally-scrollable on mobile, and contains the `VCreatorLink` and
+`VSourceLink` components.
```

Comment:
I'd recommend we implement scroll-snap here, as a nice simple enhancement for narrow desktop viewports but also the mobile experience. Example here: 

https://play.tailwindcss.com/AbfA33Za50